src/logging.ts

```diff
@@ -312,10 +310,10 @@ class LogWrapper {
     this.waitUntilResolve!()
   }
 
-  protected finishResponse(response?: Response, error?: any) {
+  protected finishResponse(response?: Response, error?: Error) {
```

Comment:
I think we could also be more explicit where the parameters are like:

```typescript
protected finishResponse(response: Response, error: undefined) {}
protected finishResponse(response: undefined, error: Error) {}
```

which forces either `response` or `error`. Right now, both are optional. Just depends on what behavior you want.